xclim/core/indicator.py

```diff
@@ -253,7 +253,9 @@ def _parse_docstring(cls, kwds):
         return kwds
 
     @classmethod
-    def _parse_cf_attrs(cls, kwds):
+    def _parse_cf_attrs(
+        cls, kwds: Dict[str, Union[str, Callable, int, bool, List[str]]]
+    ) -> Union[List[Dict[str, str]], List[Dict[str, Union[str, Callable]]]]:
```

Comment:
phew... haha That one is complex.
But I don't really understand the difference between : "List[Dict[str, str]]" and "List[Dict[str, Union[str, Callable]]]". Isn't the second only a generalization of the first?

Reply:
Yeah, this needs to be simplified. The errors being raised by mypy might be due to the fact that things being fed to this function COULD be any of these things. As we tighten the types up a bit, I imagine that this will change.

Reply:
A problem with typing these class methods is that `kwds` can contain _anything_ an indicator can be initialized with. Sure we might be able to come up with a typing compound that covers all types xclim currently uses in its indicators. But, there's not guarantee that a future (or external) subclass might use something fancy outside from those! 
Can we type with "Any"? Or ignore typing?

Reply:
You were right about this, it was nearly too much for `typing` to parse. `kwds: Dict[str, Any]` should work fine though.